src/EventBuilder.php

```diff
@@ -9,8 +9,7 @@
             $until, // string of date until event will repeat
             $maxRepeats; // integer how much maximum repeats event will occured
     
-    public $dates = [];
-    
+    public $dates = [];  
```

Comment:
__[code_sniffer]__
* Scope keyword "public" must be followed by a single space
* There must not be more than one property declared per statement